tools/rosgraph/src/rosgraph/network.py

```diff
@@ -400,11 +400,13 @@ def encode_ros_handshake_header(header):
     # in the usual configuration, the error 'TypeError: can't concat bytes to str' appears:
     if python3 == 0:
         #python 2
+        fields = [str(f) for f in fields]
```

Comment:
This is a no-op, because `f` is already a `str`, as it is the result of `"{}".format`